src/clj/rems/form_validation.clj

```diff
@@ -45,6 +45,14 @@
       {:field-id (:field/id field)
        :type     :t.form.validation/invalid-email})))
 
+(defn- invalid-phone-number-error [field]
+  (when (= (:field/type field) :phone-number)
+    (when-not (or (str/blank? (:field/value field))
+                  (re-matches +phone-number-regex+ (:field/value field)))
+      {:field-id (:field/id field)
+       :type     :t.form.validation/invalid-phone-number})))
```

Comment:
I'd like to see a test that exercises this code. Just a small new case in rems.test-form-validation is enough.

Reply:
I can do that np